src/index.ts

```diff
@@ -39,7 +39,7 @@ export function parse_config(config?: Config): Required<Config>
 
     return {
         fetch,
-        baseUrl: config?.baseUrl ?? "http://api.gamejolt.com/api/game",
+        baseUrl: config?.baseUrl ?? "https://api.gamejolt.com/api/game",
```

Comment:
Now that the major version has been bumped, I think it's a great opportunity to use the HTTPS URL by default.